src/Microsoft.Health.Dicom.Core/Features/Diagnostic/LogForwarderExtensions.cs

```diff
@@ -44,4 +46,28 @@ internal static class LogForwarderExtensions
 
         telemetryClient.TrackTrace(telemetry);
     }
+
+    /// <summary>
+    /// Emits a trace log with forwarding flag set and adds properties from instanceIdentifier as properties to telemetry.
+    /// </summary>
+    /// <param name="telemetryClient">client to use to emit the trace</param>
+    /// <param name="message">message to set on the trace log</param>
+    /// <param name="operationId">operation id</param>
+    /// <param name="input">Input payload to pass to the forward logger</param>
+    public static void ForwardLogTrace(
```

Comment:
Since we include operationId, this doesnt feel like generic method. Should we scope the method name to be like ForwardOperationLogTrace? Or do you want to include the operationId as part of the input?